commands/cloudapi-v6/image.go

```diff
@@ -761,12 +761,6 @@ func RunImageList(c *core.CommandConfig) error {
 		images = sortImagesByTime(images, viper.GetInt(core.GetFlagName(c.NS, cloudapiv6.ArgLatest)))
 	}
 
-	if itemsOk, ok := images.GetItemsOk(); ok && itemsOk != nil {
```

Comment:
I think a check for `ok == false` and`itemsOk == nil` could still be useful though. Maybe just eliminate the `len` check?